uftrace.c

```diff
@@ -130,6 +131,7 @@ __used static const char uftrace_help[] =
 "      --avg-self             Show average/min/max of self function time\n"
 "      --avg-total            Show average/min/max of total function time\n"
 "  -a, --auto-args            Show arguments and return value of known functions\n"
+"      --no-args              Do not show arguments and return value\n"
```

Comment:
I know why you placed it here, but I think we should maintain it in alphabetical order.

Reply:
Right. I just reordered it. Thanks.